client/components/vertical-nav/index.jsx

```diff
@@ -2,6 +2,12 @@
  * External dependencies
  */
 import React from 'react';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
```

Comment:
We agreed to split CSS Modules work into smaller PRs. It doesn't seem like we should enable styles before we finish all other required tasks.